### DISCUSSION TITLE
Honeywell RUD (4557a-rexuanz)

### DISCUSSION THREAD

**Discussion by mcattle** [bemasher/rtlamr#293] [Q&A]
Has anyone been able to get readings from a Honeywell RUD (4557a-rexuanz) two-way smart electricity meter?

[Here's](https://fccid.io/QZC-REXUANZ/User-Manual/Manual-2738132.pdf) what I've been able to dig up on the communications chip:

**Device specifications**  
Table 1:  900 MHz Radio Specifications 

Property | Value | -
---|---|---
Classification |  Frequency Hopping Spread Spectrum
Maximum Output Power  | 250 mW
Frequency Band | 902.4 - 927.6 MHz
Number of Channels  | 25
Channel Spacing | 400 kHz
Mode  | EA LAN1 | EA LAN2 
Data Rate | 35.5 kbps | 142 kbps 
20 dB Occupied Bandwidth | 250 KHz < BW < 400 KHz | 250 KHz < BW < 400 KHz 
99% Occupied Bandwidth | 250 KHz < BW < 400 KHz | 250 KHz < BW < 400 KHz 
Max channel dwell time |  < 0.4 seconds within a 10 second period  |
Classification | Frequency Hopping Spread Spectrum 
Maximum Output Power | 1 W 
Frequency Band | 902.4 - 927.6 MHz 
Number of Channels | 64 
Channel Spacing | 400 KHz 
Mode | NGC 
Data Rate | 50 kbps, 150 kbps or 200 kbps 
20 dB Occupied Bandwidth | 100 KHz < BW < 400 KHz for 50/150/200 kbps data rates 
99% Occupied Bandwidth | 100 KHz < BW < 400 KHz for 50/150/200 kbps data rates 
Max channel dwell time | < 0.4 seconds within a 20 second period

**Comment by s0ph0s-dog** [comment]
I am also interested in whether anyone has done this successfully. Toronto Hydro has installed one of these on the unit I'm renting, and I would very much like to be able to track power usage at the meter without going to stick a camera or some similar sensor to it.

**Comment by mfreymond** [comment]
I would also like this. 

I have a Neptune R900 water meter that I am able to read, but the 4556-REXUANZ power meter isn't showing up.

Also located in Ontario, East of Toronto.

I do see an occasional SCM+ message, but after 12+ hours of logging there's only 4 unique MeterID's.  Also after 12+ hours of logging, I've only captured 2 SCM (non-plus) messages

If helpful to the author, I can capture some raw audio samples.
